integ/resources/cfn-kinesis-s3-firehose.yml

```diff
@@ -42,8 +42,18 @@ Resources:
     Type: AWS::Kinesis::Stream
     Properties:
       ShardCount: 5
+      StreamEncryption:
+        EncryptionType: KMS
+        KeyId: alias/aws/kinesis
```

Comment:
Have double checked for the keyId and put the official link there if someone else has similar concern about the keyId value is correct or not: https://docs.aws.amazon.com/kinesis/latest/APIReference/API_StartStreamEncryption.html

Reply:
Thanks